pkg/controller/direct/directbase/directbase_controller.go

```diff
@@ -223,7 +233,7 @@ func (r *reconcileContext) doReconcile(ctx context.Context, u *unstructured.Unst
 			return false, nil
 		}
 		if k8s.HasFinalizer(u, k8s.DeletionDefenderFinalizerName) {
-			// deletion defender has not yet been finalized; requeuing
+			// deletion defender has not yet finalized; requeuing
 			logger.Info("deletion defender has not yet been finalized; requeuing", "resource", k8s.GetNamespacedName(u))
```

Comment:
Nit: I agree `not yet finalized` is clearer; want to change the logger.Info message also?